### PR TITLE
feat: handle faqs with markdown

### LINKED ISSUE #56: [Filler Task]: Check if markdown it's necessary instead of having like constants

### PR DESCRIPTION
**Pull request by VmMad**
Fixes #56

### CODE REVIEW

File: src/lib/utils.ts [hunk 1]
```diff
@@ -111,3 +111,32 @@ export function formatDateToDDMMYYYY(date?: Date | null) {
         new Date(date)
     );
 }
+
+/**
+ * Parses the HTML string and returns an array of sections.
+ * @param html - The HTML string to parse.
+ * @returns An array of sections.
+ */
+export function parseMarkdownToFaqs(html: string): FaqSection[] {
```
Comment (begonaalvarezd) [inline src/lib/utils.ts:120]: we could try having this function a bit more generic so h2 & h3 are not the only headings allowed. You have an example in the assembly repo 🙏🏼 https://github.com/boxfish-studio/assembly-website/blob/main/src/components/indexed-section.svelte#L36

**Review by begonaalvarezd — CHANGES_REQUESTED** [review]
Code looks amazing great job!
Havent tested it yet though
Added a mini comment

**Review by begonaalvarezd — CHANGES_REQUESTED** [review]
Testing the code I saw that not all FAQs are rendered, some are empty. 
Please review that everything displays like in production 🙏🏼 

image
image
image
image

**Comment by evavirseda** [comment]
> Testing the code I saw that not all FAQs are rendered, some are empty. Please review that everything displays like in production 🙏🏼
> 
> image image image image

I had the same experience, and I think this could be resolved by removing the `else` and making sure that the level is `>= firstHeadingLevel`. That way, we'll have the first heading included as well. However, we can remove it later. I hope this helps :smiley: :pray:

**Review by begonaalvarezd — APPROVED** [review]

**Pull request merged**